app/views/posts/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<section>
+  <h2>New Post</h2>
+  <% if @post.errors.any? %>
+    <div>
+      <% @post.errors.full_messages.each do |message| %>
+      <p><%= message %></p>
+      <% end %>
+    </div>
+  <% end %>
+  <%= form_with model: @post, url: user_posts_path(@user) do |form| %>
+    <div>
+      <%= form.label :title, class: "form-label" %>
+      <%= form.text_field :title, required: true, autofocus: true, class: "form-input" %>
+    </div>
+    <div>
+      <%= form.label :text, class: "form-label" %>
+      <%= form.text_area :text, required: true, autofocus: true, class: "form-input" %>
+    </div>
+    <%= form.submit "Submit", class: "form-button" %>
+  <% end %>
```

Comment:
- [ ] @harshitajain06 Great work so far, however, am unable to add a post using your form, kindly ensure the user is able to add a post using this form.
- [ ] Also add a way to add like to a post, this is a project requirement 
![image](https://user-images.githubusercontent.com/61408860/230726157-a59c5478-c4c2-4226-b2c4-e4d23b8a96e3.png)